spec/08-pattern-matching.md

```diff
@@ -240,18 +240,22 @@ the pattern.
 An extractor pattern cannot match the value `null`. The implementation
 ensures that the `unapply`/`unapplySeq` method is not applied to `null`.
 
+The type `O` is said to be an _extractor type_ for some type `T` if it has a
```

Comment:
```suggestion
A type is said to be an _extractor type_ for some type `T` if it has a
```
Since the name `O` is not actually used elsewhere, maybe remove it to avoid complex name resolution for the reader.